lib/internal/source_map/prepare_stack_trace.js

```diff
@@ -106,7 +106,7 @@ const prepareStackTrace = (globalThis, error, trace) => {
         }
       }
     } catch (err) {
-      debug(err.stack);
+      debug(err.message);
```

Comment:
```suggestion
      debug(err);
```

This should print the whole error in case debug mode is active while not slowing anything down in case debug mode is not active (it would be slower when debug is active).

If that's favorable, please replace all entries accordingly.